Update debug configuration to match latest PTVSD debug configuration

Broken test due to upstream `PTVSD` changes:
Note: This isn't a bug in PTVSD. These tests have been designed to ensure VSC is in sync with PTVSD, and now its time to sync.

```js
1) Debugging - Capabilities
       Compare capabilities:
      AssertionError: expected { Object (supportsModulesRequest, supportsConfigurationDoneRequest, ...) } to deeply equal { Object (supportsExceptionInfoRequest, supportsConfigurationDoneRequest, ...) }
      + expected - actual
             "filter": "uncaught"
             "label": "Uncaught Exceptions"
           }
         ]
      -  "supportTerminateDebuggee": true
         "supportsConditionalBreakpoints": true
         "supportsConfigurationDoneRequest": true
         "supportsEvaluateForHovers": true
         "supportsExceptionInfoRequest": true
      
      at Object.<anonymous> (src/test/debugger/capabilities.test.ts:99:45)
      at Generator.next (<anonymous>)
      at fulfilled (out/test/debugger/capabilities.test.js:6:58)
      at <anonymous>
```

--- a/news/3 Code Health/1719.md
+++ b/news/3 Code Health/1719.md
@@ -1,0 +1,1 @@
+Update debug capabilities to add support for the setting `supportTerminateDebuggee` due to an upstream update from [PTVSD](https://github.com/Microsoft/ptvsd/issues).

--- a/src/client/debugger/mainV2.ts
+++ b/src/client/debugger/mainV2.ts
@@ -70,6 +70,7 @@ export class PythonDebugger extends DebugSession {
         body.supportsHitConditionalBreakpoints = true;
         body.supportsSetExpression = true;
         body.supportsLogPoints = true;
+        body.supportTerminateDebuggee = true;
         body.exceptionBreakpointFilters = [
             {
                 filter: 'raised',

